### ISSUE TITLE
No longer working on Debian 11 because of glibc version

### ISSUE THREAD

**Issue by faulpeltz** [TryGhost/node-sqlite3#1663]
### Issue Summary

/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /builds/.../node_modules/sqlite3/lib/binding/napi-v6-linux-glibc-x64/node_sqlite3.node)

### Steps to Reproduce

node-sqlite3 stopped working withe the current 5.1.3 release on latest debian stable (11-bullseye) and Ubuntu 20.04 (focal-fossa),
both only have glibc 2.31 installed (but both are stable/LTS and still supported for a long time)

Not sure if this is intended? The ci build uses Ubuntu 22 which has a newer glibc version

### Version

5.1.3

### Node.js Version

18.12.1

### How did you install the library?

npm on Debian 11

**Comment by chgeo** [comment]
Can confirm, same issue for me after upgrading to 5.1.3

Process-wise, it would be really helpful if such changes would not be done in a patch version of `sqlite3`.  I would have expected at last a minor release here.

**Comment by Hiryus** [comment]
Hello,

I have the same problem with official docker images.
Tested on image `node:16` and `node:16-bullseye-slim`.

It worked fine with sqlite3 version 5.1.2 but version 5.1.3 throws the following error:
```
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tests/node_modules/sqlite3/lib/binding/napi-v6-linux-glibc-x64/node_sqlite3.node)
```

**Comment by svetbg** [comment]
> Can confirm, same issue for me after upgrading to 5.1.3
> 
> If this GLIBC bump is intended and not a bug, it would be really helpful if such changes would not be done in a patch version of `sqlite3`. I would have expected at last a minor release here.

+1

**Comment by daniellockyer** [comment]
Hi all!

I am aware of this issue - currently testing CI on Ubuntu 20 and then I'll do another release soon 🙂

**Comment by maxballenger** [comment]
Hi, thanks for fixing. It looks like I'm getting hit by this in my project, but I'm not sure exactly how. I am pinned to an older version. This is what my yarn.lock has:

sqlite3@^5.0.2:
  version "5.0.8"
  resolved "https://registry.yarnpkg.com/sqlite3/-/sqlite3-5.0.8.tgz#b4b7eab7156debec80866ef492e01165b4688272"
  integrity sha512-f2ACsbSyb2D1qFFcqIXPfFscLtPVOWJr5GmUzYxf4W+0qelu5MWrR+FAQE1d5IUArEltBrzSDxDORG8P/IkqyQ==
  dependencies:
    "@mapbox/node-pre-gyp" "^1.0.0"
    node-addon-api "^4.2.0"
    tar "^6.1.11"
  optionalDependencies:
    node-gyp "8.x"
I installed at about 14:35 GMT. Then I got this when trying to run on Ubuntu 18.04:

Error: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found
I'm thinking this will resolve soon as everything propagates through the pipelines, but I'm confused as to how I ended up getting a glibc 2.33 binary in the first place when I supposedly downloaded version 5.0.8.

Thanks!

**Comment by maxballenger** [comment]
OK it looks like I likely have some kind of problem with a different version of `sqlite3` being installed by some other package manager - I think the problem is on my end.

**Comment by daniellockyer** [comment]
The issue should now be fixed in v5.1.4. Apologies for that - it seems `ubuntu-latest`i in GHA updated to Ubuntu 22.04 since the last release of node-sqlite3, and I missed it during the prebuild steps.

**Comment by Hiryus** [comment]
I confirm it works. 👏
Thank you for your reactivity.
